src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="product-card" data-qa="card">
+    <div class="product-card__computer-view view">
+      <img
+        class="view"
+        src="images/imac.jpeg"
+        alt="Apple computer"
+      >
+    </div>
+
+    <div class="product-card__computer-name name">
+    Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+    </div>
```

Comment:
```suggestion
    <h2 class="product-card__computer-name name">
      Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
    </h2>
```
use semantic tags - it is heading (`h2` for example)